server/methods/eraseRoom.js

```diff
@@ -17,7 +17,7 @@ Meteor.methods({
 			});
 		}
 
-		if (RocketChat.authz.hasPermission(fromId, `delete-${ room.t }`, rid)) {
+		if (RocketChat.authz.hasPermission(fromId, `delete-${ room.t }`, rid)||automatic===true) {
```

Comment:
This is a HUGE security issue.

Anyone logged can call `Meteor.call('eraseRoom', 'some rid', true)` to delete any channel even without permission